gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.html

```diff
@@ -22,20 +22,20 @@
         <div class="details-card__header__info-inputs__first-row">
           <mat-form-field class="details-card__header__info-inputs__first-row__name-field">
             <mat-label>Application name</mat-label>
-            <input formControlName="name" matInput type="text" maxlength="50" required />
+            <input formControlName="name" matInput type="text" maxlength="50" required [readonly]="isReadOnly" />
```

Comment:
I think it's better to use the `disabled` state provided by the form control in these situations.

Reply:
see how it's done in the other forms ;)

Reply:
Done

Reply:
haha no  📲